arviz/plots/backends/bokeh/traceplot.py

```diff
@@ -62,9 +62,17 @@ def plot_trace(
         **backend_kwargs,
     }
     dpi = backend_kwargs.pop("dpi")
+    labels = [
+        make_label(var_name, selection) for idx, (var_name, selection, value) in enumerate(plotters)
+    ]
```

Comment:
You need to have the same names than those that go into the CDS. And that tooltip should change based on the axis.

And In traceplot the x is `index` (this could be draw or ndraw * chain + draw) and y is the value of the parameter.

Reply:
Could you please tell me what ndraw is?

Reply:
number of draws

Reply:
By the same "names" do you mean the python variable names like cds_name instead of var_name?

Reply:
Ah never mind I think I understand now.

Reply:
Are the names like "theta_t_ARVIZ_CDS_SELECTION_school_Choate" ?

Reply:
yes.